src/components/ReportActionItem/ReportActionItemImages.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import RenderHTML from '../RenderHTML';
+import stylePropTypes from '../../styles/stylePropTypes';
+import Text from '../Text';
+import Image from '../Image';
+
+const propTypes = {
+    /** array of image and thumbnail URIs */
+    images: PropTypes.arrayOf(
+        PropTypes.shape({
+            thumbnail: PropTypes.string,
+            image: PropTypes.string,
+        }),
+    ).isRequired,
+
+    /** optional: max number of images to show in the row if different than images length */
+    // eslint-disable-next-line react/require-default-props
```

Comment:
We should add defaults for these instead of disabling the lint rule.

Reply:
The goal is to have size and total get their "default" value from the image prop length, but you can't set default props from other props AFAIK. This was the best I could come up with, and I can add a comment to note this. [Context](https://expensify.slack.com/archives/C05DWUDHVK7/p1692046582429169?thread_ts=1691789223.761189&cid=C05DWUDHVK7)

Reply:
@Li357 Tagging for visibility